pmd-java/src/main/resources/category/java/design.xml

```diff
@@ -813,14 +813,19 @@ in each object at runtime.
  [@Final=true() and @Static=false()]
  [not(preceding-sibling::Annotation/MarkerAnnotation/Name[@Image="Builder.Default"]
     and //ImportDeclaration/Name[@Image="lombok.Builder"])]
+ [not(ReferenceType)]
 /VariableDeclarator
  [VariableInitializer/Expression/PrimaryExpression[not(PrimarySuffix)]
-  /PrimaryPrefix/*
+  /PrimaryPrefix[not(./AllocationExpression/ClassOrInterfaceType[pmd-java:typeIs('java.lang.Thread')])]
```

Comment:
Is this change related (excluding java.lang.Thread)? If so, please add a test case